### ISSUE TITLE
What is the ETA on the code

### ISSUE THREAD

**Issue by MrigankRaman** [pratyushasharma/laser#2]
Thanks for the awesome work. I would like to try it out. What is the ETA on the code?

**Comment by dkmisra** [comment]
A development release is available now. We will release a major refactoring in the first half of January to make the code very easy to apply to a new domain in BigBench and other popular LLM benchmark. 

If you have any questions about using the development, then please let us know. For now, there is a file for each LLM and dataset combination and you just need to pip install the requirements and then run these files and providing the lname, lnum, and rate hyperparameters. The only place which will need some work for some files is that you need to do hyperparameter selection on the validation set separately.

**Comment by dkmisra** [comment]
Closing this thread. Feel free to re-open the thread. We welcome you to checkout the refactored version in mid-January, and if you have questions about this developmental version then either raise a new issue or email us. Thanks.
